emailsender/pkg/metrics/metrics.go

```diff
@@ -0,0 +1,54 @@
+// Package metrics implements Prometheus metrics for email sender service
+package metrics
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	prometheusNamespace = "acs"
+	prometheusSubsystem = "emailsender"
+	clusterIDLabelName  = "cluster_id"
+)
+
+var (
+	metrics *Metrics
+	once    sync.Once
+)
+
+// Metrics holds the prometheus.Collector instances
+type Metrics struct {
+	emailsSent *prometheus.CounterVec
+}
+
+// Register registers the metrics with the given prometheus.Registerer.
+func (m *Metrics) Register(r prometheus.Registerer) {
+	r.MustRegister(m.emailsSent)
+}
+
+// IncEmailsSent increments the metric counter for started probe runs.
+func (m *Metrics) IncEmailsSent(clusterID string) {
+	m.emailsSent.With(prometheus.Labels{clusterIDLabelName: clusterID}).Inc()
+}
+
+// NewInstance returns the global Singleton instance for Metrics.
+func NewInstance() *Metrics {
```

Comment:
I would remove the "New" prefix here, because it implies that this will always create a New Instance which it will not.
```suggestion
func NewInstance() *Metrics {
```